pkgs/applications/editors/emacs/elisp-packages/manual-packages/beancount-mode/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  trivialBuild,
+  fetchFromGitHub,
+  emacs,
+  lib,
+}:
```

Comment:
This format is not consistent with nixpkgs.  Put them in one line or write something like this:

```
{ lib,
, trivialBuild
, fetchFromGitHub
, emacs
}:
```